packages/grid/src/GridRenderer.ts

```diff
@@ -2352,7 +2354,8 @@ export class GridRenderer {
       }
     );
 
-    context.translate(0, gridY);
+    // Now move to the edge of the "grid" in the so we can draw the contents of the dragging column in the clipped region
```

Comment:
Not sure if this is what you meant, but looks like something is missing in the comment

```suggestion
    // Now move to the edge of the "grid" in the clipped region so we can draw the contents of the dragging column in the clipped region
```